API-strategie-modulen/API-strategie-mod-geo/crs.md

```diff
@@ -123,7 +126,7 @@ The CRS can be specified for request and response individually using parameters
   <uL>
     <li>Issue an HTTP GET request to the API, including the <code>bbox</code> parameter AND the <code>bbox-crs</code> parameter.</li>
     <li>Validate that a document was returned with a status code 200.</li>
-    <li>Verify that the geometries in the document have the coordinate reference system that was requested using the <code>bbox-crs</code> parameter.</li>
+    <li>Verify that the response includes a <code>Content-Crs</code> http header with the value of the requested CRS identifier.</li>
```

Comment:
Om het specifieker te maken zou je hier kunnen toevoegen dat in de Content-Crs header de URI van de gevraagde CRS teruggegeven moet worden als waarde.

Reply:
done